nestkernel/node_manager.cpp

```diff
@@ -724,14 +720,12 @@ void
 NodeManager::finalize_nodes()
 {
 #ifdef _OPENMP
-  LOG( M_INFO, "NodeManager::finalize_nodes()", " using OpenMP." );
-// parallel section begins
 #pragma omp parallel
   {
     index t = kernel().vp_manager.get_thread_id();
 #else
-    for ( index t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
-    {
+  for ( index t = 0; t < kernel().vp_manager.get_num_threads(); ++t )
+  {
```

Comment:
You managed to trick `clang-format`: it thinks these two lines should be indented one more level. This is why TravisCI complains. Either you indent them, or surround them with `// clang-format off` and `// clang-format on` or we need to think off something else (clang-format-3.8 still has the same problem).